lib/PuppeteerSharp/Target.cs

```diff
@@ -41,7 +48,10 @@ internal Target(Browser browser, TargetInfo targetInfo)
         /// Gets the type. It will be <see cref="TargetInfo.Type"/> if it's "page" or "service_worker". Otherwise it will be "other"
         /// </summary>
         /// <value>The type.</value>
-        public string Type => _targetInfo.Type == "page" || _targetInfo.Type == "service_worker" || _targetInfo.Type == "browser" ? _targetInfo.Type : "other";
+        public string Type => _targetInfo.Type == "page" || _targetInfo.Type == "service_worker" || _targetInfo.Type == "browser" ?
```

Comment:
is it too late to introduce an enum for this?

Reply:
It's never too late :)